inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -211,10 +211,10 @@ DEFINE_string(layout, "", layout_message);
 DEFINE_int32(qb, 16, gna_qb_message);
 
 /// @brief Specify precision for all input layers of the network
-DEFINE_string(ip, "", inputs_precision_message);
+DEFINE_string(ip, "U8", inputs_precision_message);
```

Comment:
It is not backward compatible behavior. Because 'U8" was used by default for image inputs only. Also, it affects the Workbench where benchmark_app is used. This can break non-vision topologies, where a higher range of input data required.

Reply:
Reverted and updated the condition on `Configuring inputs and outputs` stage. Please take a look one more time.